typescript/utils/src/logging.ts

```diff
@@ -1,14 +1,7 @@
-import { Console } from 'console';
-
+/* eslint-disable no-console */
 type LOG_LEVEL = 'none' | 'error' | 'warn' | 'info' | 'debug' | 'trace';
 
-const console = new Console({
-  stdout: process.stdout,
-  stderr: process.stderr,
-  groupIndentation: 4,
-});
-
-const consoleLog = console.log.bind(console);
+const consoleLog = console.log;
```

Comment:
I'd also be up to just get rid of this variable and use `console.log` directly, but don't feel strongly